### PR TITLE
chore: polishing readme

### PR DESCRIPTION
**Pull request by wey-gu**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] doc

#### Description:

Revisiting and polishing readme

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [x] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by Sophie-Xie — DISMISSED** [review]

File: README-CN.md [hunk 1]
```diff
@@ -30,9 +30,8 @@
 
 **NebulaGraph** 是一款开源的图数据库，擅长处理千亿个顶点和万亿条边的超大规模数据集。
 
-已陆续被包括在 **BAT**, **TMD** 中的众多知名大厂和大量国内外中小企业使用，<br />
-广泛应用在社交媒体、实时推荐、网络安全、金融风控、知识图谱、人工智能等。<br />
-https://nebula-graph.com.cn/cases
+在被[众多社区用户](https://nebula-graph.com.cn/cases)采用、共建、与演进中，[NebulaGraph 社区](https://github.com/vesoft-inc/nebula-community/)被广泛应用、验证在社交媒体、实时推荐、网络安全、金融风控、知识图谱、人工智能等大规模生产场景，并且成长为一个活跃的开源社区，拥有[丰富的生态周边项目](https://docs.nebula-graph.com.cn/master/20.appendix/6.eco-tool-version/)，欢迎从[nebula-graph.com.cn/cases](https://nebula-graph.com.cn/cases)了解我们的用户案例。
```
Comment (randomJoe211) [inline README-CN.md:33]: 1. “社区被演进”、“社区被应用在xx场景”和“社区拥有周边项目”的用法不符合语言逻辑和中文语法规范
2. 演进和成长意义重复，可以简化
3. 一句话用了两边cases的链接，体验不好也没必要

一种建议的改法：
 NebulaGraph 社区已成长为一个荟聚了众多用户、融合了各类图技术场景实践知识的活跃开源社区。你可以在其中与大家共同交流 NebulaGraph [周边生态项目](https://docs.nebula-graph.com.cn/master/20.appendix/6.eco-tool-version/)的应用心得，或者社交媒体、实时推荐、网络安全、金融风控、知识图谱、人工智能等[大规模生产场景](https://nebula-graph.com.cn/cases)的实践经验。

File: README-CN.md [hunk 2]
```diff
@@ -45,13 +44,17 @@ https://nebula-graph.com.cn/cases
 
 ## 发布通告
 
-v1.x 和 v2.5.x 之后的版本，NebulaGraph 在这个 repo 管理。<br />
-如需获取 v2.0.0 到 v2.5.x 之间的版本，请访问 [NebulaGraph repo](https://github.com/vesoft-inc/nebula-graph)。
+NebulaGraph 的 GitHub 仓库经历过拆分和合并的过程，所以其 v2.0.0 到 v2.5.x 的代码分布在另外几个仓库中：[Nebula-Graph](https://github.com/vesoft-inc/nebula-graph)、[Nebula-Storage](https://github.com/vesoft-inc/nebula-storage)、[Nebula-Common](https://github.com/vesoft-inc/nebula-common)，这几个仓库将被归档。
+
+从 v2.6.0 开始，包括现在的 v3.x，NebulaGraph 内核代码都在单独的当前 [github.com/vesoft-inc/nebula](https://github.com/vesoft-inc/nebula) 下。
```
Comment (randomJoe211) [inline README-CN.md:49]:
```suggestion
从 v2.6.0 开始，包括现在的 v3.x，NebulaGraph 内核代码集中在 [github.com/vesoft-inc/nebula](https://github.com/vesoft-inc/nebula) 下。
```

确定要写”包括现在的 v3.x“吗，以后每次换大版本都要改这一句

File: README.md [hunk 3]
```diff
@@ -38,17 +35,21 @@ Compared with other graph database solutions, **NebulaGraph** has the following
...
-* Role-based access control for higher level security
+* Role-based access control for higher-level security
 
 ## Notice of Release
 
-This repository hosts the source code of NebulaGraph versions before 2.0.0-alpha and after v2.5.x.  <br/>
-If you are looking to use the versions between v2.0.0 and v2.5.x, please head to [NebulaGraph repo](https://github.com/vesoft-inc/nebula-graph).
+NebulaGraph used to be split into three repositories: [Nebula-Graph](https://github.com/vesoft-inc/nebula-graph), [Nebula-Storage,](https://github.com/vesoft-inc/nebula-storage) and [Nebula-Common](https://github.com/vesoft-inc/nebula-common) for versions between v2.0.0 and v2.5.x, which will be archived.
+
+It's back to mono-repo since v2.6.0 and v3.x here at [github.com/vesoft-inc/nebula](https://github.com/vesoft-inc/nebula).
```
Comment (randomJoe211) [inline README.md:44]: "since v2.6.0 and v3.x" is logically incorrect. Delete "and v3.x" or rephrase the sentence.

File: README.md [hunk 4]
```diff
@@ -38,17 +35,21 @@ Compared with other graph database solutions, **NebulaGraph** has the following
...
+
+Please check the latest released version via the documentation: https://docs.nebula-graph.io/.
```
Comment (randomJoe211) [inline README.md:46]: The word "latest" is an adjective and can be used to modify the verb "release". "the latest release" or "the latest version" would be enough.

File: README-CN.md (hunk 1 above)
Comment (wey-gu) [reply README-CN.md:33]: Thanks, @randomJoe211! will fix all your suggestions accordingly later :D

Comment (wey-gu) [reply README-CN.md:33]: perfect, adopted with your suggestion

File: README.md (hunk 3 above)
Comment (wey-gu) [reply README.md:44]: thanks! fixed!

**Review by randomJoe211 — APPROVED** [review]

**Review by Sophie-Xie — APPROVED** [review]

**Pull request merged**